numpy/ma/core.py

```diff
@@ -6959,7 +6959,7 @@ def where(condition, x=_NoValue, y=_NoValue):
         element from `x`, otherwise from `y`.
     x, y : array_like, optional
         Values from which to choose. `x` and `y` need to have the same shape
-        as condition, or be broadcast-able to that shape.
+        as condition, or be broadcastable to that shape.
```

Comment:
"have the same shape" is a subset of "be broadcastable to that shape", so arguably isn't needed